docs/reference/commandline/dockerd.md

```diff
@@ -1231,10 +1231,12 @@ The `--metrics-addr` option takes a tcp address to serve the metrics API.
 This feature is still experimental, therefore, the daemon must be running in experimental
 mode for this feature to work.
 
-To serve the metrics API on localhost:1337 you would specify `--metrics-addr 127.0.0.1:1337`
-allowing you to make requests on the API at `127.0.0.1:1337/metrics` to receive metrics in the
+To serve the metrics API on localhost:9323 you would specify `--metrics-addr 127.0.0.1:9323`
+allowing you to make requests on the API at `127.0.0.1:9323/metrics` to receive metrics in the
 [prometheus](https://prometheus.io/docs/instrumenting/exposition_formats/) format.
 
+Note that port 9323 is the [default port associated](https://github.com/prometheus/prometheus/wiki/Default-port-allocations) with Docker metrics to avoid collisions with other prometheus exporters and services.
```

Comment:
Can you wrap this to ~80 chars? Also, I think we can drop the "Note" (we generally try to reserve "Note" for things to call out specifically). Something like;

```suggestion
Port `9323` is the [default port associated with Docker
metrics](https://github.com/prometheus/prometheus/wiki/Default-port-allocations)
to avoid collisions with other prometheus exporters and services.
```